norddrop/src/device/types.rs

```diff
@@ -87,21 +52,43 @@ pub enum FinishEvent {
     },
 }
 
-#[derive(serde::Serialize)]
+// TODO(msz): make timestamp consistent across events
+// and make use of `SystemTime` since uniffi supports it
```

Comment:
Breakage of swig API. I choose to care about this since I find it easier to split the work into multiple stages. I'll remove swig bindings in the next PR

Reply:
Yeah, the `device` layer and C layer since they are somehow mixed together